app/src/main/res/values/strings.xml

```diff
@@ -278,7 +278,7 @@
 	<string name="exclude_nsfw_from_history_summary">Manga marked as NSFW will never be added to the history and your progress will not be saved</string>
 	<string name="clear_cookies_summary">Can help in case of some issues. All authorizations will be invalidated</string>
 	<string name="show_all">Show all</string>
-	<string name="invalid_domain_message">Invalid domain</string>
+	<string name="invalid_domain_message">Invalid server address</string>
```

Comment:
This string is also used for manga source domains so it would be better to create a new string resource

Reply:
I`ll do it